ajax/libs/jquery.rondell/1.1.0/jquery.rondell.css

```diff
@@ -0,0 +1,670 @@
+.rondell-container {
+  overflow: hidden;
+  display: block;
+  position: relative;
+  font-size: 12px;
+}
+.rondell-container.rondell-initializing {
+  background: url("images/loader.gif") no-repeat 50% 50%;
```

Comment:
I think you still need to add the `images` folder because `jquery.rondell.css` need the files under `images`.